kiwi/storage/subformat/__init__.py

```diff
@@ -56,9 +58,8 @@ def new(
             'base': 'Base'
         }
         try:
-            (custom_args, module_namespace) = DiskFormat._custom_args_for_format(
-                name, xml_state
-            )
+            (custom_args, module_namespace) = DiskFormat.\
+                _custom_args_for_format(name, xml_state)
```

Comment:
Just out of curiosity. Is this chance done for any specific reason? I am fine with it, just curious.

Reply:
Only to break up the long line. Do you see an issue with it ?

Reply:
No, not at all. It was just curiosity to know if there was any further motivation beyond breaking the line.